share/global/global.go

```diff
@@ -95,31 +77,33 @@ func SetGlobalObjects(rtSocket string, regResource RegisterDriverFunc) (string,
 	return platform, flavor, cloudPlatform, network, containers, nil
 }
 
-func getContainerPlatform(c *container.ContainerMeta) string {
+// Get the container platform and the cloud provider in getContainerPlatform
+func getContainerPlatform(c *container.ContainerMeta) (string, string) {
+	var platform, cloudPlatform string
```

Comment:
platform is not default to share.PlatformDocker anymore ?

Reply:
I will fix this, thanks for the noticed